assignment/scripts/array-practice.js

```diff
@@ -31,14 +31,14 @@ console.log('--- 3. Accessing items in an array ---');
 console.log('First animal is', animalArray[0])
 
 // 3.a. TODO: Log the second animal in the array 
-
+console.log('Second animal is', animalArray[1]);
 
 // 3.b. TODO: Log the last animal in the array using it's array index 
-
+console.log('Third anmial is', animalArray[3]);
 
 // 3.c. (STRETCH) TODO: Log the last animal by using the array length, 
 //      instead of the exact index number of the last item
-
+console.log('Third animal is', animalArray[animalArray.length-1]);
```

Comment:
Often people will put spaces around operators like `-`.
```suggestion
console.log('Third animal is', animalArray[animalArray.length - 1]);
```